src/ops/noop.cc

```diff
@@ -13,38 +18,41 @@
  * limitations under the License.
  */
 
-#include "flexflow/ops/noop.h"
+// It is very unfortunate that we have to include model.h before noop.h here.
```

Comment:
What's the problem you're running into here? I didn't notice this issue with `Linear`